<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f7fafc;
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(158, 158, 158);
border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="bt_close_main">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/cerrar.svg</normaloff>iconos/cerrar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:20px;
margin-top:8px;
color: #FFFFFF;</string>
        </property>
        <property name="plainText">
         <string>Robot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_minimize_main">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/minimizar.svg</normaloff>iconos/minimizar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="Bateria">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="textobateria">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>Bater√≠a Rpi :</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="bateriaRpi">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>7</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>11</y>
           <width>21</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>52</y>
         <width>201</width>
         <height>401</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>imagenes/robImagen.jpg</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="cabezaPushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Cabeza</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bocaPushButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Boca</string>
       </property>
      </widget>
      <widget class="QPushButton" name="manoDerechaPushButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>370</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Mano
Derecha</string>
       </property>
      </widget>
      <widget class="QPushButton" name="manoIzquierdaPushButton">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>370</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Mano
Izquierda</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brazoIzquierdoPushButton">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>170</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Brazo
Izquierdo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="brazoDerechoPushButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>170</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Brazo
Derecho</string>
       </property>
      </widget>
      <widget class="QPushButton" name="caderaPushButton">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>273</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Cadera</string>
       </property>
      </widget>
      <widget class="QPushButton" name="anilloLedsPushButton">
       <property name="geometry">
        <rect>
         <x>205</x>
         <y>273</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 35px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Anillo
Leds</string>
       </property>
      </widget>
      <widget class="QPushButton" name="apagado">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>iconos/apagado.png</normaloff>iconos/apagado.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="testPrueba">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>380</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px; 
    background-color: #3498DB; 
    color: white;    
}

QPushButton:hover {
    background-color: #21618C;
}
</string>
       </property>
       <property name="text">
        <string>Test de prueba</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
